apps/ensrainbow/Dockerfile.data

```diff
@@ -49,18 +38,8 @@ WORKDIR /app/apps/ensrainbow
 COPY --from=db-data /app/ens_names.sql.gz .
 COPY --from=db-data /app/THE_GRAPH_LICENSE.txt .
 
-# produce the data dir artifact & cleanup
-RUN pnpm run ingest && rm ens_names.sql.gz
-
-###################
-## Output image
-###################
-
-FROM app-deps AS app
-
-# cwd to ensrainbow
-WORKDIR /app/apps/ensrainbow
+# produce the data dir artifact
+RUN pnpm run ingest
 
-# copy data dir from ingest
-COPY --from=ingest /app/apps/ensrainbow/data ./data
-COPY --from=ingest /app/apps/ensrainbow/THE_GRAPH_LICENSE.txt ./data
+# cleanup input file to reduce image size
+RUN rm ens_names.sql.gz
```

Comment:
We could create the final stage from `node-20:alpine` so that we drop all the `node_modules` and whatnot from the last stage, copying only the ingest data dir artifact.

Thoughts on that, @shrugs?

Reply:
that makes sense 👍